### ISSUE TITLE
Unable to parse this address

### ISSUE THREAD

**Issue by nishantpant** [MysteryDash/AddressParser#1]
Trying to parse this. The locality comes back as NULL. The locality should be the city. In this case Joppa.
Any ideas?

521 Terrapin Terrace, Joppa, MD 21085, USA

**Comment by MysteryDash** [comment]
Indeed, that's unexpected considering that the address is full. Let met check that out, I'll come back to you.

**Comment by MysteryDash** [comment]
For some reason it wasn't working until I updated the library to .NET Standard 2.0. I'm going to push the update here on Nuget.

**Comment by nishantpant** [comment]
Thanks so much. Totally appreciate your help with this.
